src/aiovodafone/api.py

```diff
@@ -209,19 +210,28 @@ async def _get_page_result(self, page: str) -> dict[Any, Any]:
         _LOGGER.debug("GET reply %s: %s", page, reply_json)
         return await self._list_2_dict(reply_json)
 
-    async def get_user_data(self) -> dict[Any, Any]:
+    async def get_sensor_data(self) -> dict[Any, Any]:
         """Load user_data page information."""
         _LOGGER.debug("Getting sensor data for host %s", self.host)
 
-        reply_dict = await self._get_page_result("/data/user_data.json")
-        return reply_dict
+        reply_dict_1 = await self._get_page_result("/data/user_data.json")
+        reply_dict_2 = await self._get_page_result("/data/statussupportstatus.json")
+        reply_dict_3 = await self._get_page_result("/data/statussupportrestart.json")
```

Comment:
Merging the data together seems like it makes it harder to handle.

If you want to return it in a single call, I would wrap it in a `dataclass` with 4 values

Reply:
Each call returns a `dict` with few values.
Using such single `dict` in HomeAssistant integration makes it easy to access info without the complexity to know where those are originally stored.

Simone

Reply:
From an HA perspective, how you implement in the lib doesn't really matter so if thats the way you want to do it that is OK